mtags/src/main/scala-2/scala/meta/internal/pc/completions/MatchCaseCompletions.scala

```diff
@@ -430,4 +529,92 @@ trait MatchCaseCompletions { this: MetalsGlobal =>
     }
   }
 
+  object CaseExtractors {
+    object CaseDefMatch {
```

Comment:
Could you add examples of matched code to each case?

Reply:
There are code examples above every case, should I add some more? `CaseDefMatch` doesn’t have any, because it's not actually a case, just a suffix of path in every other case

Reply:
Ach, alright! I missed that somehow, sorry!